src/app/zones/locales/en-us/index.yaml

```diff
@@ -0,0 +1,105 @@
+zones:
+  form:
+    exit: 'Exit'
+    nameLabel: 'Name'
+    createZoneButtonLabel: 'Create Zone & generate token'
+    environmentLabel: 'Environment'
+    universalLabel: 'Universal'
+    kubernetesLabel: 'Kubernetes'
+    zoneIngressLabel: 'Zone Ingress'
+    zoneIngressEnabledLabel: 'Enabled'
+    zoneEgressLabel: 'Zone Egress'
+    zoneEgressEnabledLabel: 'Enabled'
+    connectZone: 'Connect Zone'
+    scan:
+      waitTitle: 'Waiting for Zone to be connected …'
+      completeTitle: 'Done!'
+      completeDescription: 'The Zone “{name}” is now connected.'
+      completeButtonLabel: 'Go to Zone “{name}”'
+      errorTitle: 'Could not connect Zone'
+      errorDescription: 'We were unable to connect Zone.'
+    universal:
+      copySaveToken:
+        title: 'Copy and save token'
+        alertMessage: 'You won’t see the token again! Make sure to save it somewhere now.'
+      connectZone:
+        title: 'Connect Zone'
+        description1: 'Copy and paste the following configuration into the config.yaml on your local machine.'
+        fileName: 'config.yaml'
+        config: |
+          environment: universal
+          mode: zone
+          multizone:
+          zone:
+            name: {zoneName}
+            globalAddress: {globalKdsAddress}
+          kuma:
+          multizone:
+            zone:
+              kds:
+                auth:
+          cpTokenInline: {token}
```

Comment:
**Issue**: A few indentation fixes needed here as well. Note that I kept the `kuma` key even though that one will definitely not exist in Kuma; it just makes it easier to see where this part would be in other places.

See https://github.com/kumahq/kuma/blob/master/docs/generated/kuma-cp.md for reference.

```suggestion
          environment: universal
          mode: zone
          multizone:
            zone:
              name: {zoneName}
              globalAddress: {globalKdsAddress}
          kuma:
            multizone:
              zone:
                kds:
                  auth:
                    cpTokenInline: {token}
```